src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/DepthPerSampleHC.java

```diff
@@ -53,7 +53,7 @@ public void annotate( final ReferenceContext ref,
         Utils.nonNull(gb);
 
         if ( likelihoods == null || !g.isCalled() ) {
-            logger.warn("Annotation will not be calculated, genotype is not called or alleleLikelihoodMap is null");
+            logger.warn(AnnotationUtils.generateMissingDataWarning(vc, g, likelihoods));
```

Comment:
I think the `OneShotLogger` should be separate from the normal logger in this class, with a clear name like `oneShotLogger`, so that people don't mistakenly try to write additional messages to the `OneShotLogger` and wonder why they don't show up.